src/main/java/seedu/address/logic/commands/calendar/CalCommand.java

```diff
@@ -14,6 +14,8 @@
 public abstract class CalCommand extends Command {
 
     public static final List<String> ALL_COMMAND_WORDS = List.of(COMMAND_WORD_VIEW, COMMAND_WORD_FIND);
+    public static final String CAL_VIEW_FORMAT = COMMAND_GROUP_CAL + " " + COMMAND_WORD_VIEW + " this/week";
```

Comment:
So now view is showing this or any week ah. Or is this typo.

Reply:
yeah it's a typo